exercise2/main.go

```diff
@@ -42,10 +47,6 @@ func main() {
 	// - Feel free to create a new function. Stand for the DRY (Don't Repeat Yourself) principle
 }
 
-func MoveTheCar(c Car, length int) {
```

Comment:
There are two functions that actually have the same pattern. They receive 2 params and both call the move function. 

So the solution is, make an interface to abstraction this function. And make a new function that using the new interface 

We can make a new interface that represents both of the structs for this PR I call `Mover`. So we don't have two functions `MoveTheCar` and `MoveTheBicycle` that has the same functionality.